x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/header.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { EuiFlexGroup, EuiFlexItem, EuiTabs, EuiTab, EuiSpacer } from '@elastic/eui';
+import { Props as EuiTabProps } from '@elastic/eui/src/components/tabs/tab';
+
+const Container = styled.div`
+  border-bottom: ${props => props.theme.eui.euiBorderThin};
+  background-color: ${props => props.theme.eui.euiPageBackgroundColor};
+`;
+
+const Wrapper = styled.div`
+  max-width: 1200px;
```

Comment:
For the max size, can we use a constant? there is nothing in the theme for that?

Reply:
not in the theme unfortunately, the closest is `EuiPage` `restrictWidth` property, but that defaults to `1000px`